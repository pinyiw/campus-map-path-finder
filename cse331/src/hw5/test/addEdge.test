# Create a graph
CreateGraph graph

# Add nodes
AddNode graph a
AddNode graph b
AddNode graph c

# Add edges
AddEdge graph a b e1
AddEdge graph b c e2
AddEdge graph c a e3