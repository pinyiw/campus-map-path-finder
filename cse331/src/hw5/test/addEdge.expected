# Create a graph
created graph graph

# Add nodes
added node a to graph
added node b to graph
added node c to graph

# Add edges
added edge e1 from a to b in graph
added edge e2 from b to c in graph
added edge e3 from c to a in graph